Formula/pachi.rb

```diff
@@ -33,13 +33,9 @@ def install
     system "make"
     bin.install "pachi"
 
-    if build.with? "patterns"
-      share.install resource("patterns")
-    end
+    share.install resource("patterns") if build.with? "patterns"
```

Comment:
Not sure this shouldn't be `pkgshare` really, although that's not changed by your rearranging.